src/Sylius/Bundle/CoreBundle/Fixture/Factory/CatalogPromotionExampleFactory.php

```diff
@@ -144,4 +138,13 @@ private function getLocales(): iterable
             yield $locale->getCode();
         }
     }
+
+    private function getDate(\DateTimeInterface|string $date): \DateTimeInterface
+    {
+        if ($date instanceof \DateTimeInterface) {
+           return $date;
```

Comment:
Missing one space before `return` 😄 but this whole function won't be needed after the first comment is fixed 🖖